src/main/java/com/tickets/service/mapper/impl/OrderMapperImpl.java

```diff
@@ -0,0 +1,27 @@
+package com.tickets.service.mapper.impl;
+
+import com.tickets.model.Order;
+import com.tickets.model.Ticket;
+import com.tickets.model.dto.OrderResponseDto;
+import com.tickets.service.mapper.OrderMapper;
+import java.time.format.DateTimeFormatter;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.springframework.stereotype.Service;
+
+@Service
```

Comment:
```suggestion
@Component
```
this applies to all mappers